browser/main/modals/PreferencesModal/UiTab.js

```diff
@@ -252,6 +253,16 @@ class UiTab extends React.Component {
               {i18n.__('Show a confirmation dialog when deleting notes')}
             </label>
           </div>
+          <div styleName='group-checkBoxSection'>
+            <label>
+              <input onChange={(e) => this.handleUIChange(e)}
+                checked={this.state.config.ui.showDate}
+                ref='showDate'
+                type='checkbox'
+              />&nbsp;
+              {i18n.__('Show date information about a note in side panel')}
```

Comment:
The message is not really clear to me. Maybe something like: `Show full date information in note list`? I'm bad at naming stuff :smile:

Reply:
I agree with you. I myself have had some doubts while trying to get a name for that option :sweat_smile: But the one you suggested seems to be just fine to me. I'll rename it.